elasticsearch.md

```diff
@@ -525,7 +525,13 @@ In ReadonlyREST we roughly classify requests as:
 * "write": when allowed, the request changes the internal state of the cluster or the data.
 
 If a **read request** asks for a some indices they have permissions for and some indices that they do NOT have permission for, the request is **rewritten** to involve only the subset of indices they have permission for.
-This is behaviour is very useful in Kibana: **different** users can see the **same** dashboards with data from only their own indices.|
+This is behaviour is very useful in Kibana: **different** users can see the **same** dashboards with data from only their own indices.
+
+When the subset of indices is empty, it means that user are not allowed to access requested indices. In multitenancy environment we should consider two options:
+* requested indices don't exist
+* requested indices exist but belong to another user  
+
+For both these cases ROR is going to return HTTP 404 or empty response. The same behaviour will be observed for ES with ROR disabled (for nonexistent index). If some index does exist, but belongs to another user, ROR is going to pretend that the index doesn't exist and the response is the same like the index actually did not exist. 
```

Comment:
* indices belonging to someone -> someone being authorized to see some indices
* say what is the response when the index actually does not esist.